python/pyspark/mllib/clustering.py

```diff
@@ -95,6 +106,13 @@ def predict(self, x):
                 best_distance = distance
         return best
 
+    def computeCost(self, rdd):
+        """Return the K-means cost (sum of squared distances of points to their nearest center) for this
```

Comment:
python style (newline for long doc strings + shorter doc lines <= 72 chars in length following PEP 8):
```
"""
Return the K-means cost (sum of squared distances of points to their
nearest center) for this model on the given data.
"""
```